src/bloom.cpp

```diff
@@ -53,6 +54,33 @@ void CBloomFilter::setup(unsigned int nElements,
     nFlags = nFlagsIn;
 }
 
+void CBloomFilter::setup_guaranteeFPR(unsigned int nElements,
```

Comment:
nit: we normally CamelCase style for function/method name

Reply:
Done.